### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by cloudbehl**
tendrl-bug-id's: #660, #659, #658, #654, #651

### CODE REVIEW

File: src/modules/bricks/host-bricks/host-bricks.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h5 ng-if="vm.brickList.length !== 0">{{vm.filteredbrickList.length}}
                     {{brick.port}}
                 </div>
                 <div class="ft-column ft-actions">
-                    <button class="btn btn-default launch-btn" ng-click="vm.redirectToGrafana(brick, $event)" uib-tooltip="Launch Dashboard" tooltip-placement="auto bottom-left">Launch Dashboard</button>
+                    <button class="btn btn-default launch-btn" ng-click="vm.redirectToGrafana(brick, $event)" tooltip-placement="auto bottom-left">Launch</button>
```
Comment (a2batic) [inline src/modules/bricks/host-bricks/host-bricks.html]: @cloudbehl, according to https://github.com/Tendrl/ui/issues/651, 'Launch Dashboard' -> 'Dashboard'

File: src/modules/bricks/volume-bricks/volume-bricks.html [hunk 2]
```diff
@@ -107,7 +107,7 @@ <h5 ng-if="vm.subVolumeList.length !== 0">{{vm.totalBrick}}
                         <div class="list-group list-view-pf list-view-pf-view " ng-repeat="brick in subVolume.bricks | orderBy: 'fqdn'">
                             <div class="list-group-item">
                                 <div class="list-view-pf-actions">
-                                    <button class="btn btn-default" ng-click="vm.redirectToGrafana(brick, $event)">Launch Dashboard</button>
+                                    <button class="btn btn-default" ng-click="vm.redirectToGrafana(brick, $event)">Launch</button>
```
Comment (a2batic) [inline src/modules/bricks/volume-bricks/volume-bricks.html]: @cloudbehl, according to https://github.com/Tendrl/ui/issues/651, 'Launch Dashboard' -> 'Dashboard'

**Review by a2batic — APPROVED** [review]

File: src/commons/stores/node-store.js [hunk 3]
```diff
@@ -95,6 +95,7 @@
                     host.role = store.findRole(list[i].tags).role;
                     host.integrationId = list[i].cluster.integration_id;
                     host.managed = _getManagedState(clusters, host);
+                    host.alerts = list[i].alerts ? list[i].alerts : 0;
```
Comment (gnehapk) [inline src/commons/stores/node-store.js]: @cloudbehl If the alerts are not available, then we should display "No Data" as per comment on https://github.com/Tendrl/ui/issues/658.

File: src/commons/stores/volume-store.js [hunk 4]
```diff
@@ -46,7 +46,7 @@
                     temp.clusterId = data[i].cluster_id;
                     temp.rebalStatus = data[i].rebal_status;
                     temp.brickCount = data[i].brick_count;
-                    temp.alertCount = data[i].alert_count;
+                    temp.alertCount = data[i].alert_count ? data[i].alert_count : 0;
```
Comment (gnehapk) [inline src/commons/stores/volume-store.js]: @cloudbehl same as above comment.

File: src/modules/bricks/host-bricks/host-bricks.html [hunk 5]
```diff
@@ -74,7 +74,7 @@ <h5 ng-if="vm.brickList.length !== 0">{{vm.filteredbrickList.length}}
                     {{brick.port}}
                 </div>
                 <div class="ft-column ft-actions">
-                    <button class="btn btn-default launch-btn" ng-click="vm.redirectToGrafana(brick, $event)" uib-tooltip="Launch Dashboard" tooltip-placement="auto bottom-left">Launch Dashboard</button>
+                    <button class="btn btn-default launch-btn" ng-click="vm.redirectToGrafana(brick, $event)" tooltip-placement="auto bottom-left">Dashboard</button>
```
Comment (gnehapk) [inline src/modules/bricks/host-bricks/host-bricks.html]: @cloudbehl as uib-tooltip is removed, then you have to remove uib-placement as well.

**Comment by gnehapk** [comment]
@cloudbehl  I think you have added #656 issue by mistake as I don't see any changes for that.

**Comment by cloudbehl** [comment]
@gnehapk yes, will amend the PR

**Review by gnehapk — APPROVED** [review]

**Pull request merged**